docs/en/customization/middleware.md

```diff
@@ -1,19 +1,36 @@
 # Customization — HTTP Middleware
 
-## Video tutorial
+RoadRunner provides a flexible and extensible architecture that allows developers to build custom middleware for both
+`HTTP` and `gRPC` plugins. Moving highly loaded parts of an application, such as authentication, to middleware written
```

Comment:
`gRPC` not middleware, but interceptors (client side). And also you might mention `temporal` here (also called interceptors).